### ISSUE TITLE
3d export of tree model is apparently backwards in the z-order.

### ISSUE THREAD

**Issue by JLOrwell** [xLightsSequencer/xLights#4233]
**Describe the bug**
This is easy to reproduce with a 180 tree.  If you export and import the same model, the new model will be reversed in z-order.  I do not know for sure it's the exporter that is backwards.  It may be the importer, however if i look at the xmodel text in the file, the first layer has the farthest section of the tree and this is backwards from how the custom model data is displayed.

In my test, I had a 1500 node tree set to string length 1500, 15 strands, 180 tree.

Windows 10
xlights version is 2023-22

**Comment by cybercop23** [comment]
Exported MT (left) 180, imported on right, 3d... seems okay.. See attached. Please 
[Mega Tree Small.xmodel.zip](https://github.com/xLightsSequencer/xLights/files/14135446/Mega.Tree.Small.xmodel.zip)

**Comment by JLOrwell** [comment]
That isn't a custom model.  I guess I didn't specify but the problem is specifically when you export as a custom model.

**Comment by cybercop23** [comment]
ahh.. this can't be right... "xlights version is 2015-22" ?
The import of a Custom model had the starting location lower left, just like the original model. The Wiring view is by default backwards, so is that maybe what you looked at?
Also, why would you export a native model as a custom model?
[Tree-2.xmodel.zip](https://github.com/xLightsSequencer/xLights/files/14216071/Tree-2.xmodel.zip)

**Comment by JLOrwell** [comment]
I don't know why my reason would matter but there are plenty of reasons to do so, which is why the option is there to do so.  In my particular case I have a different program that can import them.
And yes i'm sure about the wiring diagram.  In this photo the tree on the left is the original and the right is the result of exporting and importing again.  You can see in both cases the start is the bottom left.
![image](https://github.com/xLightsSequencer/xLights/assets/12838841/20be4660-f63b-4365-85ef-61cef5e48aa3)

**Comment by cybercop23** [comment]
Good luck

**Comment by JLOrwell** [comment]
you were right.  I somehow posted my version wrong.  I am using 2023.22 on the computer i just reproduced this on.

**Comment by JLOrwell** [comment]
@AzGilrock I did give the reason if you were paying attention.  I never claimed or even acted like I didn't want to say why.  The reason is irrelevant though.  If you have the option to do it, it should work properly.  If you must know so much about my proprietary software, I'll tell you.  I wrote a program capable of exporting and importing 2d and 3d xlights models and editing them because the built-in custom model editor is a piece of crap.  It's so hard to use that I wrote a completely separate program to do it in that allows rotating, zooming, skewing, etc., and I'm not the first person to write my own model importer.  So don't you feel better about yourself now?  You guys need to change how you think about your bugs.  "why would anyone ever use the feature we just added" is pretty dumb.
